guides/common/modules/proc_using-vmware-cloud-init-and-userdata-templates-for-provisioning.adoc

```diff
@@ -83,6 +88,8 @@ datasource:
     seedfrom: https://{foreman-example-com}/userdata/
 EOM
 ----
++
+If you intend to provision through {SmartProxy}, use the URL of your {SmartProxy} in the `seedfrom` option, such as `http://_{smartproxy-example-com}_:8000/userdata/`.
```

Comment:
Use "SmartProxyServer" here?

Also, use HTTPS in the example?

Reply:
Good catch, thanks!